native/src/sagas/watchFetchCategory.ts

```diff
@@ -21,45 +18,28 @@ import { LanguageResourceCacheStateType } from '../redux/StateType'
  * @param routeCity The key of the current route
  * @returns true if the fetch corresponds to a peek
  */
-function* isPeeking(routeCity: string): Generator<SelectEffect, void> {
-  yield select(state =>
+function* isPeeking(routeCity: string): SagaGenerator<boolean> {
+  return yield* select(state =>
```

Comment:
:thinking:

Reply:
before we actually did call isPeeking, but it would always return `undefined`, of course, as return was missing. That's also what you see in the tests. If a route isPeeking the languages are not fetched for example.

Reply:
Yes, was just surprised that we did not return it, so the logic was faulty all the time :see_no_evil: